src/main/java/org/sciborgs1155/robot/Ports.java

```diff
@@ -7,19 +7,23 @@ public static final class OI {
     public static final int DRIVER = 1;
     public static final int LEFT_STICK = 2;
     public static final int RIGHT_STICK = 3;
+
+    public static final int LEFT_JOYSTICK = -1;
+    public static final int RIGHT_JOYSTICK = -1;
```

Comment:
I'm not exactly sure why these exist, `LEFT_STICK` and `RIGHT_STICK` refer to our joysticks